bin/fin

```diff
@@ -6377,8 +6414,9 @@ case "$1" in
 		elif [[ "$1" == "--project-images" ]]; then
 			update_project_images
 		elif [[ "$1" == "--self" ]]; then
-			echo "Downloading $URL_FIN"
-			curl -kfsSL "$URL_FIN?r=$RANDOM" | sudo tee "$FIN_PATH_UPDATED" >/dev/null
+			fin_file_url=$(get_latest_version_fin)
```

Comment:
The variable where `get_latest_version_fin` result is stored is called `fin_url` in the `check_for_updates` function, for consistency it should be called the same here.